db/seeds.rb

```diff
@@ -43,7 +44,8 @@
   reason_to_donate: 'I want to help a girl receive an excellent education',
   slug: 'annie-wright-schools',
   info_url: 'http://www.aw.org',
-  donation_url: 'https://aw.myschoolapp.com/Page/Support/Make-a-Gift?siteId=693&ssl=1'
+  donation_url: 'https://aw.myschoolapp.com/Page/Support/Make-a-Gift?siteId=693&ssl=1',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.